lg02-pkt-fwd/src/lg02_single_rx_tx.c

```diff
@@ -240,7 +243,7 @@ int main(int argc, char *argv[])
     loradev->invertio = 0;
     strcpy(loradev->desc, "RFDEV");	
 
-    MSG("Radio struct: spi_dev=%s, spiport=%d, freq=%ld, sf=%d, bw=%ld, cr=%d, pr=%d, wd=0x%2x, power=%d\n", radio, loradev->spiport, loradev->freq, loradev->sf, loradev->bw, loradev->cr, loradev->prlen, loradev->syncword, loradev->power);
+    MSG("JSON-forwarding Radio struct: spi_dev=%s, spiport=%d, freq=%ld, sf=%d, bw=%ld, cr=%d, pr=%d, wd=%d (0x%2x), power=%d\n", radio, loradev->spiport, loradev->freq, loradev->sf, loradev->bw, loradev->cr, loradev->prlen, loradev->syncword, loradev->syncword, loradev->power);
```

Comment:
You're sure we cannot get NULL reference exceptions here?

Reply:
I am sure; either the settings are filled in correctly a couple of lines above it, or (when neither 'device 1' nor 'device 2' is specified) the program already borks earlier, because the GPIO-pins cannot be set properly.

Reply:
... but why not actually add a proper check for this variable? :upside_down_face: 


Done!